bin/oneoff/pullOutBlocks.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/node
+
+/*
+  This was a one off script used to create .json files for each block in App Lab and Game Lab.
+  The json files will be used to seed ProgrammingExpressions.
+ */
+
+var fs = require("fs");
+
+var applabBlocksToImport = [
+  // Add dropletConfig definition of the blocks you want to import here
+];
+
+var gamelabBlocksToImport = [
+  // Add dropletConfig definition of the blocks you want to import here
+];
+
+applabBlocksToImport.forEach(block => {
+  console.log(JSON.stringify(block));
+  fs.writeFile(
+    `../../dashboard/config/programming_expressions/applab/${block.func}.json`,
+    JSON.stringify(block),
```

Comment:
@dmcavoy to pretty print, you would do: 
```suggestion
JSON.stringify(block, null, 2)
```

Reply:
Used this to pretty print everything. 

```
function readFiles(dirname, onFileContent, onError) {
  fs.readdir(dirname, function(err, filenames) {
    if (err) {
      onError(err);
      return;
    }
    filenames.forEach(function(filename) {
      fs.readFile(dirname + filename, "utf-8", function(err, content) {
        if (err) {
          onError(err);
          return;
        }
        onFileContent(filename, content);
      });
    });
  });
}
var data = {};
readFiles(
  "dashboard/config/programming_expressions/spritelab/",
  function(filename, content) {
    //data[filename] = content;
    let blockInfo = JSON.parse(content);
    fs.writeFile(
      `dashboard/config/programming_expressions/spritelab/${filename}`,
      JSON.stringify(blockInfo, null, 2),
      function(err) {
        if (err) {
          throw err;
        }
        console.log(
          `Saved! dashboard/config/programming_expressions/spritelab/${filename}`
        );
      }
    );
  },
  function(err) {
    throw err;
  }
);
```